ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,658 @@
+     *
+     * @requires  modules/drilldown
+     *
+     * @function Highcharts.Chart#multipleDrillUp
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     * @param {Highcharts.Breadcrumbs} drillAmount
+     *        A number of drillups that needs to be performed.
+     */
+    public multipleDrillUp(this: Breadcrumbs, drillAmount: number): void {
+        const chart = this.chart,
+            breadcrumbsList = this.breadcrumbsList;
+
+        if (breadcrumbsList && breadcrumbsList.length) {
+            for (let i = 0; i < breadcrumbsList[breadcrumbsList.length - 1][0] - drillAmount; i++) {
+                chart.drillUp();
+            }
+        }
+
+        this.destroy(drillAmount);
+    }
+    /**
+    * Redraw.
+    *
+    * @requires  modules/drilldown
+    *
+    * @function Highcharts.Chart#redraw
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    */
+    public redraw(this: Breadcrumbs): void {
+        const breadcrumbsGroup = this.breadcrumbsGroup;
+
+        if (breadcrumbsGroup) {
+            this.alignGroup();
+        }
+    }
+
+    /**
+    * Render the button.
+    *
+    * @requires  modules/drilldown
+    *
+    * @function Highcharts.Chart#renderButton
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    * @param {Highcharts.Breadcrumbs} breadcrumb
+    *        Current breadcrumb
+    * @param {Highcharts.Breadcrumbs} posX
+    *        Initial horizontal position
+    * @param {Highcharts.Breadcrumbs} posY
+    *        Initial vertical position
+    * @return {SVGElement|void}
+    *        Returns the SVG button
+    */
+    public renderButton(this: Breadcrumbs, breadcrumb: Array<any>, posX: number, posY: number): SVGElement|void {
+        const breadcrumbs = this,
+            chart = this.chart,
+            breadcrumbsList = breadcrumbs.breadcrumbsList,
+            breadcrumbsOptions = chart.options.drilldown?.breadcrumbs,
+            lang = chart.options.lang;
+
+        if (breadcrumbsOptions) {
+            const button = chart.renderer.button(
+                breadcrumbsOptions.formatter ? breadcrumbsOptions.formatter(breadcrumbs) : void 0 ||
+                    format(breadcrumbsOptions.format as any,
+                        { value: breadcrumb[1].name || lang?.mainBreadCrumb }, chart),
+                posX,
+                posY,
+                function (): void {
+                    if (breadcrumbsOptions.events && breadcrumbsOptions.events.click) {
+                        breadcrumbsOptions.events.click(button as any, breadcrumbs as any);
+                        // Prevent from click on the current level
+                    } else if (breadcrumbsList[breadcrumbsList.length - 1][1].name !== button.textStr) {
+                        breadcrumbs.multipleDrillUp(breadcrumb[0]);
+                    }
+                })
+                .attr({
+                    padding: 3
```

Comment:
It should be configurable

Reply:
I hardcoded this value, to make it visually acceptable for the PoC. I would like to hear the @TorsteinHonsi opinion about the overall design before I implement this.